.storybook/preview-head.html

```diff
@@ -0,0 +1,2 @@
+<link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet">
+<link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
```

Comment:
Anything within storybook, won't be available to the UI Kit consumer, it's a dev package. If we want to use FontAwesome it needs to be imported in the `index.js`

Reply:
FA is used only in storybook. UI Kit is universal and can accept any classname to show icon